From 34f98fb6bcd790adcc9d9ba104e56d2eae908036 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@apache.org>
Date: Mon, 6 Jan 2014 23:21:08 +0000
Subject: [PATCH 1534/1612] MAPREDUCE-3310. Addendum fixing javadocs warnings (tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1556063 13f79535-47bb-0310-9956-ffa450edef68

Conflicts:
	hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapred/JobConf.java
	hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/Job.java
	hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/JobContext.java
---
 .../java/org/apache/hadoop/mapred/JobConf.java     |    5 ++---
 .../main/java/org/apache/hadoop/mapreduce/Job.java |    5 ++---
 .../org/apache/hadoop/mapreduce/JobContext.java    |    7 +++----
 3 files changed, 7 insertions(+), 10 deletions(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapred/JobConf.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapred/JobConf.java
index e3bea10..a742f56 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapred/JobConf.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapred/JobConf.java
@@ -893,7 +893,7 @@ public class JobConf extends Configuration {
    * grouping keys of inputs to the reduce.
    * 
    * @return comparator set by the user for grouping values.
-   * @see #setOutputValueGroupingComparator(Class) for details.  
+   * @see #setOutputValueGroupingComparator(Class) for details.
    */
   public RawComparator getOutputValueGroupingComparator() {
     Class<? extends RawComparator> theClass = getClass(
@@ -960,8 +960,7 @@ public class JobConf extends Configuration {
    * @param theClass the comparator class to be used for grouping keys. 
    *                 It should implement <code>RawComparator</code>.
    * @see #setOutputKeyComparatorClass(Class)
-   * @see {@link #setCombinerKeyGroupingComparator(Class)} for setting a
-   * comparator for the combiner.
+   * @see #setCombinerKeyGroupingComparator(Class)
    */
   public void setOutputValueGroupingComparator(
       Class<? extends RawComparator> theClass) {
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/Job.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/Job.java
index 1143eab..9b22415 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/Job.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/Job.java
@@ -969,7 +969,7 @@ public class Job extends JobContextImpl implements JobContext {
    * are passed to the {@link Reducer}.
    * @param cls the raw comparator
    * @throws IllegalStateException if the job is submitted
-   * @see {@link #setCombinerKeyGroupingComparatorClass(Class)}
+   * @see #setCombinerKeyGroupingComparatorClass(Class)
    */
   public void setSortComparatorClass(Class<? extends RawComparator> cls
                                      ) throws IllegalStateException {
@@ -984,8 +984,7 @@ public class Job extends JobContextImpl implements JobContext {
    *                       org.apache.hadoop.mapreduce.Reducer.Context)}
    * @param cls the raw comparator to use
    * @throws IllegalStateException if the job is submitted
-   * @see {@link #setCombinerKeyGroupingComparatorClass(Class)} for setting a
-   * comparator for the combiner.
+   * @see #setCombinerKeyGroupingComparatorClass(Class)
    */
   public void setGroupingComparatorClass(Class<? extends RawComparator> cls
                                          ) throws IllegalStateException {
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/JobContext.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/JobContext.java
index f11aa28..e182117 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/JobContext.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/JobContext.java
@@ -180,7 +180,7 @@ public interface JobContext extends MRJobConfig {
    * grouping keys of inputs to the combiner.
    *
    * @return comparator set by the user for grouping values.
-   * @see Job#setCombinerKeyGroupingComparatorClass(Class) for details.
+   * @see Job#setCombinerKeyGroupingComparatorClass(Class)
    */
   public RawComparator<?> getCombinerKeyGroupingComparator();
 
@@ -189,9 +189,8 @@ public interface JobContext extends MRJobConfig {
      * grouping keys of inputs to the reduce.
      *
      * @return comparator set by the user for grouping values.
-     * @see Job#setGroupingComparatorClass(Class) for details.
-     * @see {@link #getCombinerKeyGroupingComparator()} for setting a
-     * comparator for the combiner.
+     * @see Job#setGroupingComparatorClass(Class)
+     * @see #getCombinerKeyGroupingComparator()
      */
   public RawComparator<?> getGroupingComparator();
   
-- 
1.7.0.4

